HISTORY.rst

```diff
@@ -9,6 +9,10 @@ Unreleased Changes
   in missing datasets. Additionally added a logger ``debug`` message to note
   this "unusual" setting of these parameters in case of accidental usage
   which could be noted during development.
+* Fixed issue in zonal statistics that could cause the aggregate vector to be
+  locked due to a dereferencing race condition. This was present in some
+  cases with a flaky unit test but could have been seen in practice if the
+  vector was deleted immediately after the call ton ``zonal_statistics``.
```

Comment:
Little typo: `call ton`

Reply:
fixed in https://github.com/natcap/pygeoprocessing/pull/142/commits/4e46f37a52b3e591dd7389aa7cbede096c1cce8f